### ISSUE TITLE
searchguard 6.8.0-25.1 version issue

### ISSUE THREAD

**Issue by tmanninger** [floragunncom/search-guard#700]
Hi,

the package "6.8.0.-25.1" contains an incorrect version:

```
/usr/share/elasticsearch/bin/elasticsearch-plugin list -v
warning: Falling back to java on path. This behavior is deprecated. Specify JAVA_HOME
Plugins directory: /usr/share/elasticsearch/plugins
search-guard-6
- Plugin information:
Name: search-guard-6
Description: Provide access control related features for Elasticsearch 6
Version: 6.8.0.-25.1            <------ see here
Elasticsearch Version: 6.8.0
Java Version: 1.8
Native Controller: false
Extended Plugins: []
 * Classname: com.floragunn.searchguard.SearchGuardPlugin
```

The version of this package should be "6.8.0-25.1", but the pluginversion is "6.8.0.-25.1" (a . before the -)

Now the searchguard puppet plugin is not working correctly.

Pls fix this!

**Comment by floragunncom** [comment]
for 25.1 we can not fix it, but we will soon release 25.3 with a few bugfixes including this one

**Comment by floragunncom** [comment]
We fixed this in our build pipeline and so it will happen not again. v25.3 is imminent.
